project1/css/index.css

```diff
@@ -1,189 +1,250 @@
-  }
-  
-  h4 {
-	font-size: 18px;
-	display: inline-block;
-  }
-  
-  button {
-	font-size: 22px;
-	margin-top: 10px;
-	margin-left: 335px;
-	margin-bottom: 50px;
-	background-color: Transparent;
-	border: 1px solid grey;
-	width: 170px;
-	height: 35px;
-	color: dark grey;
-  }
-  
-  .top-content .text-content-one p {
-	width: 490px;
-	height: px;
-	font-size: 18px;
-	display: inline-block;
-	position: absolute;
-	margin-left: -65px;
-	margin-top: 28px;
-  }
-  
-  .top-content .text-content-two {
-	width: 490px;
-	height: 85px;
-	display: inline-block;
-	margin-left: 550px;
-	margin-top: -30px;
-	font-size: 17px;
-  }
-  
-  .bottom-content .text-content1 {
-	width: 300px;
-	height: 125px;
-	font-size: 17px;
-	margin-top: 200px;
-  }
-  
-  .bottom-content .text-content2 {
-	width: 300px;
-	height: 125px;
-	margin-left: 350px;
-	margin-top: -125px;
-	font-size: 17px;
-  }
-  
-  .bottom-content .text-content3 {
-	width: 300px;
-	height: 125px;
-	margin-left: 700px;
-	font-size: 17px;
-	margin-top: -125px;
-	position: absolute;
-  }
-  
-  .main-content {
-	width: 1050px;
-	height: 525px;
-	border-top: 1px solid black;
-	border-bottom: 1px solid black;
-	background-color: none;
-	margin-left: 70px;	
-	padding-top: 35px;
-	margin-bottom: 60px;
-  }
-  
-  img {
-	display: block;
-	margin-left: auto;
-	margin-right: auto;
-	width: 40%;
-  }
-  .contact {
-	font-size: 17px;
-	margin-left: 70px;
-  }
-  footer {
-	text-align: center;
-	margin: 75px;
-  }
-  
\ No newline at end of file
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: "Bangers", cursive;
+  letter-spacing: 1px;
+  margin-bottom: 10px;
+}
+
+.container{
+  border: 3px solid blue;
+  width: 1000px;
+  display:flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 20px;
+}
+
+.box1{
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: spaced-evenly;
+}
+
+a {
+  color: grey;
+  text-decoration: none; 
+}
+
+ .container header ul{
+  width: 700px;
+  height: 50px;
+  display:flex;
+  flex-wrap: nowrap; 
+  flex-direction: row;
+  justify-content: space-evenly;
+}
+
+.container header img {
+}
+
+.cta {
+  width: 875px;
+  display:flex;
+  justify-content: space-around;
+  align-items: center;
+  margin: 75px 0px;
+}
+
+.cta-text {
+  font-size: 95px;
+  text-align: center;
+}
+
+button {
+  color: ;
+  width: 175px;
+  height: 35px;
+  font-size: 20px;
+  background-color: transparent;
+  border: 1px solid black;
+}
+.main-content {
+  border-top: 1px solid grey;
+  border-bottom: 1px solid grey;
+  padding: 30px 0px;
+}
+
+.top-content {
+  display: flex;
+  width: 875px;
+  justify-content: space-evenly;
+  margin: 10px 0px;
+}
+
+.middle-img {
+  display:flex;
+  height: 150px;
+  width: 875px;
+  margin: 35px 0px;
+}
+
+.bottom-content {
+  display: flex;
+  width: 875px;
+  margin: 10px 0px;
+}
+
```

Comment:
since you know the width of all of the inner elements are the same and you want them to remain the same:
```
.top-content,
.middle-img,
.bottom-content {
  display: flex;
  width: 875;
}```
That way if you ever need to change the size you can do it quickly.
Alternatively you could set the width of the .container element.